src/btree.c

```diff
@@ -714,7 +717,9 @@ most_of_entry_is_on_left_side(uint64 total_bytes,
           < (total_bytes + sizeof(table_entry) + entry_size) / 2;
 }
 
-/* Figure out how many entries we can put on the left side.
+/*
+ * ----------------------------------------------------------------------------
```

Comment:
Few changes are like this one ... Adding banner separator lines in comment blocks.